packages/rn-tester/js/examples/ScrollView/ScrollViewExample.js

```diff
@@ -109,6 +109,14 @@ const examples = ([
       );
     },
   },
+  {
+    title: '<ScrollView> (inverted = true)\n',
+    description:
+      "You can display <ScrollView>'s child components in inverted order",
+    render: function (): React.Node {
+      return <InvertedContentExample />;
+    },
+  },
```

Comment:
Can we also put `// macOS` tags here?

Also unrelated, I kinda wanna change the macOS tag from `// TODO [(macOS GH#774)` to a simple `// [macOS` and ` // macOS]`, following what React Native Windows does. The giant TODO was because at one point we thought we might upstream all of the code. I know that's not a fun change to integrate, since it touches like 2000 files. Do you have any opinions @christophpurrer ?

Reply:
Totally happy to just use
```
// macOS
```
from now on ;-)

I already changed it from the 'older' id used  (ISS#2323203)

Feel free to do the change search&replace at any time ;-)

Reply:
Cool! Let's use the existing tag for now and maybe once we start 0.70 merge, I'll make the simplified tag change